File: backend/src/main/java/jp/co/iwaki/data/api/domain/model/resource/market/MarketItemResource.java
package jp.co.iwaki.data.api.domain.model.resource.market;

import lombok.AllArgsConstructor;
import lombok.Data;

/**
 * 市場取扱品目要素
 */
@Data
@AllArgsConstructor
public class MarketItemResource {

    /** 取り扱い量 */
    private int quantity = 0;
    /** 取り扱い円 */
    private long amount = 0L;

}

File: backend/src/main/kotlin/iwakiDataApi/resource/statistics/market/MarketItemResource.kt
package iwakiDataApi.resource.statistics.market

/**
 * 市場取扱品目要素
 */
data class MarketItemResource(
    // 取り扱い量
    val quantity : Int = 0,
    // 取り扱い円
    val amount : Long = 0
)
